about_dice_project.rb

```diff
@@ -2,9 +2,12 @@
 
 # Implement a DiceSet Class here:
 #
-# class DiceSet
-#   code ...
-# end
+class DiceSet
+  attr_reader :values
+  def roll(n)
+    @values = (1..n).map {rand(max=6) + 1}
```

Comment:
Can we use Array here?
`Array.new(size) { rand(1..6) }`

Reply:
Fixed